### PR TITLE
Add changes to listing.html make it clickable

### LINKED ISSUE #175: [Bug] Entire card is not clickable
## Description
Entire card is not clickable . since there's only one action per card the whole card should be clickable for the sake of UX , here following the[ Fitts’ law](https://www.interaction-design.org/literature/topics/fitts-law) states that the amount of time required for a person to move a pointer (e.g., mouse cursor) to a target area is a function of the distance to the target divided by the size of the target. Thus, the longer the distance and the smaller the target’s size, the longer it takes. whole clickable  card encourages interaction and reduces effort.

## Expectation
the whole card should be clickable

## Screenshots
Frame 21780
Frame 21779

-  @possumbilities @TimidRobot  I would be interested in resolving this bug.

### PR DESCRIPTION
**Pull request by NIKHIL95969**
## Fixes

Fixes #175 by @sj-jain

## Description

Not clickable full part of `listing.html` card. 

## Expectation ✅
The whole card is now clickable. 

## Screenshots

https://github.com/creativecommons/cc-resource-archive/assets/93199036/30444e87-fdad-4aec-85da-1751451c8a9d

## Checklist

- [x] My pull request has a descriptive title (not a vague title like Update index.md).
- [x]  My pull request targets the default branch of the repository (main or master).
- [x]  My commit messages follow [best practices](https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53).
- [x]  My code follows the established code style of the repository.
- [ ]  I added or updated tests for the changes I made (if applicable).
- [ ]  I added or updated documentation (if applicable).
- [x]  I tried running the project locally and verified that there are no visible errors.

## Developer Certificate of Origin
For the purposes of this DCO, "license" is equivalent to "license or public domain dedication," and "open source license" is equivalent to "open content license or public domain dedication."

<details>
  <summary>Developer Certificate of Origin</summary>
 
```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by Lakshay-Pruthi** [comment]
Thanks for contributing but I would suggest you to go through [Contributing guidelines](https://opensource.creativecommons.org/contributing-code/) first.

**Comment by possumbilities** [comment]
Hi @NIKHIL95969 thanks for the work here. 

Wrapping the entire box in the anchor is technically doable according to the `<a>` [spec](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#technical_summary): 

> Permitted content	[Transparent](https://developer.mozilla.org/en-US/docs/Web/HTML/Content_categories#transparent_content_model), except that no descendant may be [interactive content](https://developer.mozilla.org/en-US/docs/Web/HTML/Content_categories#interactive_content) or [a](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a)n a element, and no descendant may have a specified [tabindex](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex) attribute.

The use of inline styles should be avoided, can you please move them into the appropriate CSS Rules within the `style.css` file?

I'm also curious why change the `Gemfile.lock` file, seems unrelated to this specific Issue, a mistake perhaps?

**Comment by NIKHIL95969** [comment]
@possumbilities remove inline CSS into `style.css` and fix the `Gemfile.lock`

File: docs/_layouts/listing.html [hunk 1]
```diff
@@ -124,11 +124,11 @@ <h1>
 					{% if page.medium contains mediumfilter or mediumfilter == null %}
 					{% if page.language contains languagefilter or languagefilter == null %}
 						<div class="thumbnailbox {{ page.topic }} {{ page.medium }} {{page.language}}">
-						<a href="{{ page.permalink }}">
-						<div class="thumbnailtitle"><h3>{{ page.title }}</h3></div>
-						<div class="thumbnail" style="background-image: url('{{ page.image-thumb }}'); "></div>
-						</a>
-						<div class="thumbnailblurb">{{ page.blurb }}</div>
+							<a class="page_permakinik" href="{{ page.permalink }}" >
```
Comment (possumbilities) [inline docs/_layouts/listing.html:127]: did you mean for this to read `page_permalink`?

File: docs/style.css [hunk 2]
```diff
@@ -154,6 +154,10 @@ h1 {
 text-align: center;
 }
 
+.page_permakinik {
```
Comment (possumbilities) [inline docs/style.css:157]: did you mean for this to read `.page_permalink`?

**Comment by possumbilities** [comment]
@NIKHIL95969 I added two items I missed prior about the wording of the class utilized. I think that should wrap things for Approval.

**Comment by NIKHIL95969** [comment]
@possumbilities minor typo fixed check please if any

**Review by possumbilities — APPROVED** [review]
Thank you! ❤️

**Pull request merged**